src/modals/manage-farming-pool-modal/withdraw-form/use-withdraw-formik.ts

```diff
@@ -77,13 +82,13 @@ export const useWithdrawFormik = (farmId: string, farmVersion: FarmVersionEnum)
             navigateAction(ModalsEnum.Confirmation, {
               type: ConfirmationTypeEnum.InternalOperations,
               opParams,
-              testID: 'STAKE_TRANSACTION_SENT'
+              testID: 'WITHDRAW_TRANSACTION_SENT'
```

Comment:
and this selectors also

Reply:
Sorry, test IDs like these have never been specified in enums